api/Entities/FacilityEntity.cs

```diff
@@ -9,7 +9,9 @@ public class FacilityEntity
         public int Id { get; set; }
         public string Name { get; set; }
         public Realm Realm { get; set; }
-        public List<FacilityHotelEntity> FacilityHotels { get; set; } = new List<FacilityHotelEntity>();
-        public List<FacilityRoomEntity> FacilityRooms { get; set; } = new List<FacilityRoomEntity>();
+        public List<RoomEntity> Rooms { get; }
+        public List<FacilityRoomEntity> FacilityRooms { get; }
+        public List<HotelEntity> Hotels { get; }
+        public List<FacilityHotelEntity> FacilityHotels { get; }
```

Comment:
Why do we have all these properties in the Facility?

Reply:
They are used in the next pr